domains/khateeb.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "khateeboveskey"
+    },
+    "record": {
+        "CNAME": "khateeboveskey.github.io/khateeb-portfolio"
```

CNAME can't contain `/`
```suggestion
        "CNAME": "khateeboveskey.github.io"
```